apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -238,6 +249,35 @@
                 <a class="btn btn-xs btn-secondary" href="<%= urlPrefix %>/assessment/<%= assessment.id %>/manual_grading/instance_question/<%= instance_question.id %>">Manual grading</a>
                 <% } %>
               </td>
+              <td class="text-right">
+                <div class="dropdown js-question-actions">
+                  <button type="button" class="btn btn-secondary btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    Action <span class="caret"></span>
+                  </button>
+
+                  <div class="dropdown-menu dropdown-menu-right">
+                    <% if (authz_data.has_course_instance_permission_edit) { %>
+                      <button
+                        id="row<%= i_instance_question %>PopoverBreakButton"
+                        class="dropdown-item js-mark-variant-as-broken"
+                        data-toggle="popover"
+                        data-trigger="click"
+                        data-container="body"
+                        data-html="true"
+                        data-placement="auto"
+                        title="Confirm mark as broken"
+                        data-content="<%= include('popoverBreakVariant', {
+                          iRow: i_instance_question,
+                          instance_question: instance_question,
+                          assessmentInstanceOwner: assessmentInstanceOwner,
+                        }) %>"
+                      >
+                        Mark as broken
```

Comment:
I wonder whether it would be better to use some other language here, like "Generate new variant"? Or "Reset question variant"? Or even just "Reset question", but I worry people might think that also resets the score. The "broken" term has not been exposed in the UI before this PR and I think that most instructors currently think of it as forcing a question to generate a new variant -- they don't really care what we do to the old variants (marking them as broken is just our implementation detail).

In this case, "Mark as broken" is an operation on the instance question, but I think it's misleading because we are not marking the instance question as broken. If we do want to use the "broken" term then I think we should at least say "Mark existing variant as broken" or something like that. But I'd prefer to use phrasing like "reset question" or "regenerate question".

Reply:
I changed the language in 7e0b69c and 66edad4.